Web Flow With External Server Posting Back/webserver/src/oAuth.ts

```diff
@@ -37,6 +38,18 @@ export const VALID_CLIENT_TYPES : ClientType[] = ['prod','test'];
 export class OAuthClient {
     constructor(private readonly config: OAuthClientConfig) {}
 
+    getInitUrl(type: ClientType, scopes: string, state: string) : string {
```

Comment:
- [ ] Comment getInitUrl()

Reply:
State is a string here. I know that it's already been validated, but if we move the validation into this method then this method is secure.